torch/_inductor/lowering.py

```diff
@@ -690,10 +690,14 @@ def inner_fn(index):
 @register_lowering(aten._unsafe_view, type_promotion_kind=None)
 @register_lowering(aten.view, type_promotion_kind=None)
 @register_lowering(aten.reshape, type_promotion_kind=None)
-def view(x, sizes):
+def view(x, sizes_or_dtype):
+    # view could also take dtype as its parameter
+    if isinstance(sizes_or_dtype, torch.dtype):
+        return to_dtype(x, sizes_or_dtype)
```

Comment:
This is not correct semantics (and would be revealed by a test), correct dtype view semantics is bitcast:
```
In [11]: a=torch.randn(2,3)

In [12]: a
Out[12]: 
tensor([[ 1.1923, -1.2699, -0.8661],
        [ 0.7665,  0.1736,  0.9688]])

In [13]: a.view(dtype=torch.int)
Out[13]: 
tensor([[ 1066965979, -1079866187, -1084376557],
        [ 1061435139,  1043447875,  1064829971]], dtype=torch.int32)

In [14]: a.to(dtype=torch.int)
Out[14]: 
tensor([[ 1, -1,  0],
        [ 0,  0,  0]], dtype=torch.int32)
```

Reply:
Is there any util function in `prim` for this kind of conversion?

Reply:
No, I don't think so, and in any case we'd need to add triton codegen support for it.